lib/modules/manager/kustomize/extract.ts

```diff
@@ -131,6 +131,7 @@ export function extractImage(image: Image): PackageDependency | null {
       ...nameDep,
       datasource: DockerDatasource.id,
       replaceString: image.newName,
+      pinDigests: false,
```

Comment:
why not adding the autoReplaceStringTemplate from L125? Should it then support pinning?

Reply:
As I try to explain in the description, this field is not necessarily used to set the version of the image, oftentimes only the registry and/or the repository

Reply:
don't we catch the new tag or digest case before this?

I'm heavily using pinned new name in my kustomizations

Reply:
Consider this:

```yaml
images:
- name: docker.io/emissaryingress/emissary
  newName: gcr.io/datawire/emissary
```

I would not want it to become:

```yaml
images:
- name: docker.io/emissaryingress/emissary
  newName: gcr.io/datawire/emissary@sha256:...of latest
```

As the proper tag (not latest) is set somewhere else, it is only meant to change the repository